lib/parameter_parser/search_parameter_parser.rb

```diff
@@ -287,12 +287,12 @@ class Filter
     # operation is :reject or :filter
     # multivalue_query is :all or :any
     def initialize(field_name, values, operation, multivalue_query)
+      @errors = values.empty? ? [%(no value for parameter "#{field_name}")] : []
```

Comment:
When finder-frontend renders search/all it sends a request to search-api with empty params, so this line is raising a 422 when trying to load the page. 

https://github.com/alphagov/finder-frontend/blob/main/app/controllers/search_controller.rb#L28

<img width="1723" height="796" alt="Screenshot 2025-12-02 at 11 00 08" src="https://github.com/user-attachments/assets/80d28e89-1999-4f97-b9ac-5e18fcfd8982" />